MathGeoLibConfigVersion.cmake.in

```diff
@@ -0,0 +1,11 @@
+set(PACKAGE_VERSION "@FOOBAR_VERSION@")
```

Comment:
What is `@FOOBAR_VERSION@`? This is perhaps leftover typo, and was sourced from another CMake script?